roles/container_runtime/tasks/package_crio.yml

```diff
@@ -57,6 +57,12 @@
     src: crio.conf.j2
     backup: yes
 
+- name: Update crictl.yaml runtime-endpoint
```

Comment:
Where does this file come from?  Is it installed with crio?

Reply:
It is installed with cri-tools.

Reply:
On fedora, looks like this file comes from cri-o rpm.

From what I can tell, /etc/crictl.yaml is needed by the crictl cli/ kubernetes to determine where crio is listening.

We should probably keep the variable in place in crio.conf and utilize it here as well (prepend with unix://) to ensure they stay in sync.

Reply:
If 'listen' in crio.conf is set to "unix:///var/run/crio/crio.sock", it breaks crio.  The default in crio.conf is `listen = "/var/run/crio/crio.sock"`.

Reply:
yes, we just need to prepend the unix part it in crict.yaml

Reply:
The unix part needs to also go in the node-config.yaml, where l_crio_var_sock is used.  Why should we prepend the unix part in four different places so that we can variablize a default value in crio.conf?

Reply:
@mtnbikenc I forgot about the node config.

Yeah, what you're saying makes the most sense in this regard.